### PR TITLE
Setup prettier

### PR DESCRIPTION
**Pull request by cquiroz**
WDYT about using prettier to format the css/scss/less files?

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -249,10 +249,12 @@ jobs:
         uses: wearerequired/lint-action@v1
         with:
           stylelint: true
+          prettier_dir: common/src/main/webapp
```
Comment (toddburnside) [inline .github/workflows/ci.yml:252]: Isn't this file generated by sbt-lucuma now?

Comment (cquiroz) [reply .github/workflows/ci.yml:252]: Yes, that's why `build.sbt` changed in this PR

**Comment by rpiaggio** [comment]
Nice, when is this being run? I was already using the prettier VSCode plugin, but it's nice to have uniform criteria.

**Review by rpiaggio — APPROVED** [review]
LGTM

File: build.sbt [hunk 2]
```diff
@@ -334,6 +334,8 @@ def runLinters(mode: String) = WorkflowStep.Use(
   params = Map(
     "github_token"         -> "${{ secrets.GITHUB_TOKEN }}",
     "auto_fix"             -> "true",
+    "prettier"             -> "true",
+    "prettier_dir"         -> "common/src/main/webapp",
```
Comment (toddburnside) [inline build.sbt:338]: Oh. Neat!

**Review by toddburnside — APPROVED** [review]
I've been using prettier in VS Code, too. So, LGTM.

**Comment by cquiroz** [comment]
The linter just checks it. it is up to you when to run it, usually with an editor plugin

**Pull request merged**